### ISSUE TITLE
0.8.0: sphinx 4.x warnings

### ISSUE THREAD

**Issue by kloczek** [netaddr/netaddr#235]
```console
+ /usr/bin/python3 setup.py build_sphinx -b man --build-dir build/sphinx
running build_sphinx
Running Sphinx v4.1.2
making output directory... done
WARNING: html_static_path entry '_static' does not exist
building [mo]: targets for 0 po files that are out of date
building [man]: all manpages
updating environment: [new config] 13 added, 0 changed, 0 removed
reading sources... [100%] tutorial_03
/home/tkloczko/rpmbuild/BUILD/netaddr-0.8.0/netaddr/ip/__init__.py:docstring of netaddr.ip.IPAddress.__bytes__:2: WARNING: Field list ends without a blank line; unexpected unindent.
/home/tkloczko/rpmbuild/BUILD/netaddr-0.8.0/netaddr/ip/nmap.py:docstring of netaddr.ip.nmap.iter_nmap_range:6: WARNING: Inline emphasis start-string without end-string.
looking for now-outdated files... none found
pickling environment... done
checking consistency... done
writing... python-netaddr.3 { introduction installation tutorial_01 tutorial_02 tutorial_03 api changes references authors contributors copyright license } done
build succeeded, 3 warnings.
```

**Comment by jstasiak** [comment]
Hey, thank you for the reported. This has been resolved (with some delay, admittedly) in #274.

**Comment by kloczek** [comment]
Thx.
I see few other minor fixes added since last release.
Do you have any plans to flush those fixes as new release? (0.9.1?) 🤔

**Comment by jstasiak** [comment]
Yup, I expect it to be something more like 0.10.0 as there is at least one feature that I need to release and one or two tricky new fixes.
